### ISSUE TITLE
Print out changelog when updating

### ISSUE THREAD

**Issue by KyleUltimate** [theRookieCoder/ferium#49]
yeah, this will be helpful.
Also, would it be possible to NOT do this on first download, because that would certainly print out a lot of unreadable stuff.
Instead, print it when upgrading mods.

**Comment by theRookieCoder** [comment]
But this would just print out the changelog of the latest version? It wouldn't _actually_ print the changes made since the version that was previously installed. And should this be normal behaviour or an opt-in through a `--verbose` flag?

**Comment by KyleUltimate** [comment]
`--verbose`
Yes, just the latest version would be enough

**Comment by theRookieCoder** [comment]
> it would be great if this behavious can be enabled in config

Sure I'll do that then

**Comment by theRookieCoder** [comment]
> would it be possible to NOT do this on first download

How would this be done though? I guess it's the same issue with figuring out the mod from the file

**Comment by KyleUltimate** [comment]
Ah yes the age old problem
I guess it's ok to print out the first download
Since it's not that big of an issue

**Comment by theRookieCoder** [comment]
How would this be implemented for GitHub Releases? They only have the description right?

**Comment by KyleUltimate** [comment]
Yep, normally release description are served as small changelogs

**Comment by theRookieCoder** [comment]
As [this Redditor](https://www.reddit.com/r/feedthebeast/comments/umcz56/comment/i821h1v) suggested, we can also prompt the user on whether to update (like most Linux package managers) or not. And of course a `--force` option for automated upgrades
